http/cars.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
I'd like to keep this around, it's useful to show the limitations of the server. I'm all for teaching clients how to work around said limitations, but it should be in a separate file.

Someone was supposed to do a pull request in which the workaround is based on having the served page include javascript to request the car images serially. I can see us having both examples included.

Can you do a separate PR in which you show your cars.lua workaround and leave this one in place? I'd rather do one pull request per logical change, this one bundles several.

Thanks!